### ISSUE TITLE
The reproduction of the paper is completed

### ISSUE THREAD

**Issue by chengfeng0113** [xavysp/DexiNed#91]
Recently, in the research of edge detection project, the training and testing code of edge detection model is reconstructed based on [mmsegment ](https://github.com/open-mmlab/mmsegmentation)framework, which supports the reproduction of dexined and other networks. After several optimization training and fine tuning, the results on biped test set are as follows:
<img width="250" alt="ed8e8f18018ca3a6cbc00df63aa65de" src="https://user-images.githubusercontent.com/12129147/131107845-68a3f138-93b4-4e44-8f10-406cf6832714.png">
Visualization example:
![0_fused_pledge](https://user-images.githubusercontent.com/12129147/131112697-5647ba21-4789-4e46-b204-2d0dcb072b6b.jpg)
![25_fused_pledge](https://user-images.githubusercontent.com/12129147/131112721-2672c1b1-021d-4b85-b214-1e5178175080.jpg)
If you continue to fine tune the bsds500 and MDBD datasets, the visualization will be better and the edges will be thinner.
The evaluation code comes from [here](https://github.com/xavysp/DexiNed/issues/63#issuecomment-813487793), but the code does not support multi process operation and needs simple modification. Otherwise, it will take a long time to get results.
All models are trained from scratch on biped dataset without pre-training model, and online data enhancement processing is carried out.

**Comment by xavysp** [comment]
> Recently, in the research of edge detection project, the training and testing code of edge detection model is reconstructed based on [mmsegment ](https://github.com/open-mmlab/mmsegmentation)framework, which supports the reproduction of dexined and other networks. After several optimization training and fine tuning, the results on biped test set are as follows:
> <img alt="ed8e8f18018ca3a6cbc00df63aa65de" width="250" src="https://user-images.githubusercontent.com/12129147/131107845-68a3f138-93b4-4e44-8f10-406cf6832714.png">
> Visualization example:
> ![0_fused_pledge](https://user-images.githubusercontent.com/12129147/131112697-5647ba21-4789-4e46-b204-2d0dcb072b6b.jpg)
> ![25_fused_pledge](https://user-images.githubusercontent.com/12129147/131112721-2672c1b1-021d-4b85-b214-1e5178175080.jpg)
> If you continue to fine tune the bsds500 and MDBD datasets, the visualization will be better and the edges will be thinner.
> The evaluation code comes from [here](https://github.com/xavysp/DexiNed/issues/63#issuecomment-813487793), but the code does not support multi process operation and needs simple modification. Otherwise, it will take a long time to get results.
> All models are trained from scratch on biped dataset without pre-training model, and online data enhancement processing is carried out.

Hi thanks for sharing and great work, could you show us  the modification in the evaluation code please?

**Comment by chengfeng0113** [comment]
> Hi thanks for sharing and great work, could you show us the modification in the evaluation code please?

I simply added multi process support to your code, but it still took more than two hours to test on the biped test set.You can download the complete code [here](https://drive.google.com/file/d/19BKMGFSEjgEj5bAVAlPil4WIZkYbExyZ/view?usp=sharing). I'm not familiar with matlab code. It may not be modified very well, but it can really shorten the test time.You can also refer to this evaluation [code](https://github.com/xwjabc/hed/tree/master/eval).

**Comment by xavysp** [comment]
> > Hi thanks for sharing and great work, could you show us the modification in the evaluation code please?
> 
> I simply added multi process support to your code, but it still took more than two hours to test on the biped test set.You can download the complete code [here](https://drive.google.com/file/d/19BKMGFSEjgEj5bAVAlPil4WIZkYbExyZ/view?usp=sharing). I'm not familiar with matlab code. It may not be modified very well, but it can really shorten the test time.You can also refer to this evaluation [code](https://github.com/xwjabc/hed/tree/master/eval).

Thanks, I'll check on Moday.

have a great weekend
